blog/2024-04-12-pizza-and-os-talk/index.md

```diff
@@ -0,0 +1,39 @@
+---
+slug: 2024-05-12-pizza-os-talk
+title: Pizza-Powered OS Talk 
+authors: [S-Sigdel]
+tags: [open-source, kuosc, Linux, OS, pizza, KUCC]
+---
+
+
+# Pizza-Powered OS Talk
+
+## Pizza and OS Talk
+<img
+src={require('./src/img/pizza_and_os_talk.jpg').default}
+alt="pizza talk"
+height="50%" 
+width="100%"
+/>
+May-12-2024
+
+<br/>
+<br/>
+Ahh, who doesn't love to talk about OS while taking a tasty bite on a pizza? Maybe not all the people on the world, but we had an awesome opportunity to do so. This event began by bringing the orders of delicious Pizzas from "The Pizza House, Banepa" and subsequently having a gathering in the Lotus Fountain at Kathmandu University. Then, we all circled up and gave our introduction, and in-addition, what we expected from that session. Many were there for the pizza; the outcome changed however. 
+
+<br/>
+<br/>
+The talk's main objective was to distribute the ideas that we had among the people who participated. And I got to know about the different initiatives KUOSC had put forward to promote open-source. Below listed are the projects that were discussed:
+
+### Neplib Linux
+
+This is an Operating System project that KUOSC has been working on to provide to the schools of rural areas, where there hardly is infrastructure of internet. This system bundled content from Help Nepal Network for providing materials to primary and seconday level students. In addition this operating system also provides a telementry side for monitoring the computer systems deployed and maintaining them. The main objective of this project was to provide a user-friendly interfacd, specifically designed for beginners who may not have used a computer before. The operating system project intends to updte and upgrade the previously built software, bringing a fresh content to E-library deployed in remote Nepali regions. 
```

Comment:
"The operating system project intends to updte and upgrade" @S-Sigdel. can you fix the typo here

Reply:
alright it's on the way